ReactNativeClient/lib/share-handler.ts

```diff
@@ -0,0 +1,47 @@
+const Note = require('lib/models/Note.js');
```

Comment:
Could you name it "shareHandler.ts" please and import it under this name in root.js?